vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -583,6 +583,7 @@ void WindowsTextInputComponentView::unmountChildComponentView(
 void WindowsTextInputComponentView::onFocusLost() noexcept {
   Super::onFocusLost();
   sendMessage(WM_KILLFOCUS, 0, 0);
+  m_caretVisual.IsVisible(false);
```

Comment:
Does this need to be set to true when focus is gained, or does that happen automatically?

Reply:
Does the TextInput not call TxShowCaret(false) when it receives WM_KILLFOCUS?

Reply:
Did some digging, TextInput calls TxShowCaret twice when it receives WM_KILLFOCUS, once with TxShowCaret(false) and TxShowCaret(true). The callstack is the same for both calls so not sure why it's being called twice